notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -0,0 +1,773 @@
+{
```

Comment:
<p>if you provide choise which model create, you should avoid specific model name in headers and descriptions (if it describes general topic related to all models), it can mislead users in case if other then GPt-Neo model selected before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>